vck/src/commonMain/kotlin/at/asitplus/wallet/lib/data/dif/DocumentDigestEntry.kt

```diff
@@ -0,0 +1,158 @@
+@file:UseSerializers(UrlSerializer::class)
+
+package at.asitplus.wallet.lib.data.dif
```

Comment:
I think `dif` is not the right namespace

Reply:
Start `csc`: "cloud signature consortium" or `rsa`: "remote signature applications" namespace?

Reply:
Since this part in particular does not (yet?) conform to the csc standard i named the new folder `rqes` and moved the new classes there